src/main/java/com/openlattice/hazelcast/serializers/BulkEntityDataLambdasStreamSerializer.java

```diff
@@ -0,0 +1,85 @@
+package com.openlattice.hazelcast.serializers;
+
+import com.dataloom.mappers.ObjectMappers;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.Serializer;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.SetMultimap;
+import com.openlattice.authorization.serializers.EntityDataLambdasStreamSerializer;
+import com.openlattice.conductor.rpc.BulkEntityDataLambdas;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Component
+public class BulkEntityDataLambdasStreamSerializer extends Serializer<BulkEntityDataLambdas> {
```

Comment:
I think you meant to extend StreamSerializer instead of Kryo's Serializer class.

Reply:
this is using kryo-- following same model as EntityDataLambdasStreamSerializer

Reply:
Ah ok. Got it!